src/e3/anod/spec.py

```diff
@@ -387,6 +393,118 @@ def bind_to_sandbox(self, sandbox: SandBox) -> None:
+            ``PATH``. If the tool may not be found with :func:`~e3.os.fs.which`,
+            and *ldd_output* is not provided, a :exc:`FileNotFoundError`
+            exception is raised.
+
+        :param prefix: The path where to find the library to be checked.
+        :param ignored_libs: The list of additional system libraries authorized
+            in the closure.
+        :param ldd_output: An ``ldd`` command output.
+        :param case_sensitive: State if the comparison of a library name and the
+            value in *ignored_libs* should be case-sensitive or not. By default,
+            the comparison is case-sensitive.
+
+        :raise AnodError: if some of the shared libraries in *prefix* (or in
+            *ldd_output*) is not in the same directory as the analysed element.
+        :raise FileNotFoundError: if *ldd_output* is not provided and ``ldd``
+            application cannot be found in the ``PATH``.
+        """  # noqa RST304
+        ignored: list[str] = ignored_libs or []
+        errors: dict[str, list[str]] = {}
+        lib_file: str = ""
+
+        if ldd_output is None:
+            # If ldd is not in the path, return and issue a warning.
+            if not which("ldd"):
+                raise FileNotFoundError(
+                    "Shared Libraries closure check skipped: ldd could not be found."
+                )
+
+            if prefix is None:
+                prefix = self.build_space.install_dir
+
+            # Get all files matching the OS shared lib extension.
+            lib_files = find(
+                root=prefix, pattern=f"*{OS_INFO[e3.env.Env().build.os.name]['dllext']}"
+            )
+            ldd_output = e3.os.process.Run(["ldd"] + lib_files).out or ""
+        else:
+            # An ldd output has been provided.
+            lib_files = re.findall(r"^([^\t].*):$", ldd_output, flags=re.M)
+
+        if self.sandbox and hasattr(self.sandbox, "root_dir"):
+            root_dir = self.sandbox.root_dir
+        else:
+            root_dir = str(Path(self.build_space.install_dir).parent)
+
+        # Retrieve list of shared lib dependencies for sanity checking
+
+        self.log.info("Running Shared Libraries closure check")
+        # Build up a list of linked libraries. Dict key is the library
+        # name/path, value is the list of linked libraries.
+        for line in ldd_output.splitlines():
+            if line[:-1] in lib_files:
+                # Line is like ``/mypath/mydll.so:``, it's the result of
+                # ldd for ``mydll.so``.
+                lib_file = line[:-1]
+            elif lib_file and " => " in line:
+                # Line looks like:
+                # ``        otherdll.so => /other/otherdll.so (0xabcd)``
+                name, path = line.strip().split(" => ", 1)
+                path = re.sub(" (.*)", "", path)
+                # Check if we should ignore it. On Windows, the paths are
+                # case-insensitive.
```

Comment:
instead of having that comment, it think it would be better to set the default value for case_sensitive to (sys.platform == "win32")

Reply:
I had to move the platform check in the method body, or the doc would not build.